terraform/development/variables.tf

```diff
@@ -1,9 +1,9 @@
 variable "environment_name" {
-    type = string
-    default = "development"
+  type    = string
+  default = "development"
 }
 
 variable "project_name" {
-    type = string
-    default = "Housing-Search-Listener"
+  type    = string
+  default = "Housing-Search-Listener"
```

Comment:
Project name shouldbe "MTFH", could you please update it? It must have been missed before